docs/core/migration/index.md

```diff
@@ -55,7 +55,7 @@ If you get an error:
 
 > No executable found matching command dotnet-migrate
 
-Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) is only available between .NET Core CLI RC3 and 3.0.100.
+Run `dotnet --version` to see which version you are using. [`dotnet migrate`](../tools/dotnet-migrate.md) was introduced in .NET Core CLI RC3 and removed in version 3.0.100.
```

Comment:
RC3? What is RC3? If it's 1.0/1.1 and a release candidate, I would just state 1.0 (or whatever) and omit the preview/rc stuff.

Reply:
Fair point. @livarcocc can you suggest better text here?

Reply:
It was introduced in .NET Core SDK 1.0.0? I don't think we need to call out exactly the preview version where it first came out.

Reply:
Thanks, updated.